src/matching.py

```diff
@@ -388,13 +388,28 @@ class RuleStrategy:
     and they return None when there is no match
     """
 
+    @staticmethod
+    def match_verbatim(plans: list, verbatim_id: str, **kwargs):
+        """
+        Match exactly to a verbatim message's ID.
+        """
+        for plan in plans:
+            if plan["id"] == verbatim_id:
+                return {
+                    "match": verbatim_id,
+                    "confidence": 100,
+                    "plan": plan,
```

Comment:
Hmmm, I'm not a fan of calling something a plan if it's very much not a plan. I think this will be confusing down the line

Reply:
If you look at how the other methods in `RuleStrategy` work, we're using the `operation` key to denote something that is not a plan match. I'd prefer to stick with that convention rather than to overload the object that gets returned for plan matching

Reply:
So that's how I'd originally planned on doing this, along with the idea of passing in verbatims as a separate object.  However, we were then starting to mix the idea of what we have as a flag (`--parent` vs `--why-warren`) and `process_post` started to get a little bit hairy because of that.

I ended up scrapping that and going with this implementation to minimize the surface area of my first change.  I'm all for redoing it something like I'd originally thought, I'll chat with you about it.